applications/index.adoc

```diff
@@ -42,9 +42,9 @@ When the application is running, not all applications resources are used. As a c
 [id="connecting-application"]
 === Connecting an application to services
 
-An application uses backing services to build and connect workloads, which vary according to the service provider. Using the xref:../applications/connecting_applications_to_services/understanding-service-binding-operator.adoc#understanding-service-binding-operator[Service Binding Operator], as a developer, you can bind workloads together with Operator-managed backing services, without any manual procedures to configure the binding connection.
+An application uses backing services to build and connect workloads, which vary according to the service provider. Using the Service Binding Operator, as a developer, you can bind workloads together with Operator-managed backing services, without any manual procedures to configure the binding connection.
 ifndef::openshift-dedicated,openshift-rosa[]
-You can apply service binding also on xref:../applications/connecting_applications_to_services/getting-started-with-service-binding-ibm-power-ibm-z.adoc#getting-started-with-service-binding-ibm-power-ibm-z[IBM Power Systems, IBM Z, and LinuxONE environments].
+You can apply service binding also on IBM Power Systems, IBM Z, and LinuxONE environments.
```

Comment:
Now that I see this again, leaving this trailing information in the document does not make sense. I will remove the entire section from the content as it will be the best course of action for now.